Simplenote/Classes/Bundle+Simplenote.swift

```diff
@@ -2,15 +2,22 @@ import Foundation
 import UIKit
 
 
-/// Bundle: Simplenote Methods
-///
+// MARK: - Bundle: Simplenote Methods
+//
 extension Bundle {
 
     /// Returns the Bundle Short Version String.
     ///
     @objc
     var shortVersionString: String {
-        let version = infoDictionary?["CFBundleShortVersionString"] as? String
+        let version = infoDictionary?[Keys.shortVersionString] as? String
         return version ?? ""
     }
 }
+
+
+// MARK: - Private Keys
+//
+private enum Keys {
+    static let shortVersionString = "CFBundleShortVersionString"
+}
```

Comment:
`static let` for DRY and autocompletion + `enum` to collect values in a type that can't be initialized and misused + `private` because it makes sense only in the context of `Bundle` access.

![italian hand kiss gif](https://media.giphy.com/media/3o6ozkQbdfOIyCC6wU/giphy.gif)